test/Maybe.js

```diff
@@ -27,10 +35,16 @@ Maybe[FL.of] = Maybe.Just;
 
 Maybe[FL.zero] = Maybe[FL.empty];
 
-Maybe.prototype[FL.equals] = function(other) {
+function Maybe$prototype$equals(other) {
```

Comment:
For consistency with changes made in sanctuary-js/sanctuary#388 which were made for the same reason as sanctuary-js/sanctuary#359. Essentially, `Maybe a` satisfies the requirements of Setoid if `a` satisfies the requirements of Setoid.